src/main/java/se/sundsvall/supportmanagement/service/mapper/ErrandMapper.java

```diff
@@ -106,20 +104,58 @@ public static Errand toErrand(ErrandEntity entity) {
 			.orElse(null);
 	}
 
-	private static Customer toCustomer(EmbeddableCustomer customer) {
-		return Optional.ofNullable(customer)
-			.map(c -> Customer.create()
-				.withId(c.getId())
-				.withType(CustomerType.valueOf(c.getType())))
+	private static List<Stakeholder> toStakeholders(List<StakeholderEntity> stakeholderEntities) {
```

Comment:
I think that many code quality scanners (sonar, etc) recommends to return an empty list, instead of null from methods. 
This is less error prone.

Reply:
Fixed